### PR TITLE
Added Toolbar tutorial to Tool Window

### PR DESCRIPTION
**Pull request by SFC-Sarge**
@madskristensen. Updated 'custom-tool-windows.md' with the tutorial on how to add a Toolbar with buttons to the Tool Window. Please validate!

### CODE REVIEW

File: .vs/ProjectSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```
Comment (madskristensen) [inline .vs/ProjectSettings.json:3]: What is this file for?

File: .vs/docs/project-colors.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "Version": 1,
+  "ProjectMap": {
```
Comment (madskristensen) [inline .vs/docs/project-colors.json:3]: What is this file for?

**Comment by madskristensen** [comment]
This is awesome, but I don't understand all those extra files added. It seems like only the .md file needed updates, right?

**Comment by SFC-Sarge** [comment]
@madskristensen; Yes I only made changes to the md file not sure why my fork made changes to this file.  They are not required by me so please only take the md changes and then I will fetch your finial merged master.

**Comment by SFC-Sarge** [comment]
@madskristensen; I renamed my local folder then colon your master back to the fork location. and the .vs folder returned. Not sure why since I do not see that folder in your repo.  I am using the latest version of VS that updated this morning.  I hope it is not a bug in this version that changes or adds files:

image

**Comment by SFC-Sarge** [comment]
@madskristensen; I renamed my local folder then colon your master back to the fork location. and the .vs folder returned. Not sure why since I do not see that folder in your repo.  I am using the latest version of VS that updated this morning.  I hope it is not a bug in this version that changes or adds files:

image

**Comment by SFC-Sarge** [comment]
@madskristensen I meant Cloned.

**Comment by SFC-Sarge** [comment]
@madskristensen; When I show all files in Solution Explorer, I see this:

image

Note: slnx.sqlite shows changed even thou, all I did was clone the repo, then open the folder in VS and finally, Show All Files?

File: .vs/ProjectSettings.json (hunk 1 above)
Comment (Danny-McNaught) [reply .vs/ProjectSettings.json:3]: I did not change this file

File: .vs/docs/project-colors.json (hunk 2 above)
Comment (Danny-McNaught) [reply .vs/docs/project-colors.json:3]: I did not change this file

**Pull request closed without merging**